test/addons-napi/test_exception/test.js

```diff
@@ -20,9 +20,10 @@ const theError = new Error('Some error');
 
   // Test that the exception thrown above was marked as pending
   // before it was handled on the JS side
-  assert.strictEqual(test_exception.wasPending(), true,
-                     'VM was marked as having an exception pending' +
-                     ' when it was allowed through');
+  const exception_pending = test_exception.wasPending();
+  assert.strictEqual(exception_pending, true,
+                     'Expected exception to be pending, but' +
+                     `it was marked as ${exception_pending}`);
```

Comment:
I think this change might be more confusing than helpful.  Is "marked as false" going to help someone or confuse them?

How about this:

```
`Exception not pending as expected, .wasPending() returned ${exception_pending}`
```